src/Microsoft.Health.Dicom.Tests.Common/AcceptHeaderHelpers.cs

```diff
@@ -49,6 +49,6 @@ public static AcceptHeader CreateAcceptHeaderForGetFrame(string transferSyntax =
 
     public static AcceptHeader CreateAcceptHeader(string transferSyntax = "*", PayloadTypes payloadType = PayloadTypes.MultipartRelated, string mediaType = KnownContentTypes.ApplicationOctetStream, double? quality = null)
     {
-        return new AcceptHeader(mediaType, payloadType, transferSyntax, quality);
+        return new AcceptHeader(mediaType, payloadType, transferSyntax, quality.GetValueOrDefault(AcceptHeader.DefaultQuality));
```

Comment:
Would it make more sense to do the `quality.GetValueOrDefault(AcceptHeader.DefaultQuality)` inside the `AcceptHeader` constructor so that we centralize that logic?